.github/workflows/main.yml

```diff
@@ -1,21 +1,85 @@
-name: CI on Windows
+name: CI
 
 on: [push, pull_request]
 
 jobs:
-  build:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["py35", "py36", "py37", "py38", "nightly"]
+
+        include:
+          - name: "py35"
+            python: "3.5"
+            tox_env: "du12"
+
+          - name: "py36"
+            python: "3.6"
+            tox_env: "du13"
+
+          - name: "py37"
+            python: "3.7"
+            tox_env: "du14"
+
+          - name: "py38"
+            python: "3.8"
+            tox_env: "du15"
+            use_coverage: true
+
+          - name: "nightly"
+            python: "3.10-dev"
+            tox_env: "du16"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        if: matrix.python != '3.10-dev'
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Set up Python ${{ matrix.python }} (deadsnakes)
+        uses: deadsnakes/action@v1.0.0
+        if: matrix.python == '3.10-dev'
```

Comment:
Could you use `endsWith` function like README of deadsnakes/action does? I'll add `3.9-dev` later.

Reply:
makes sense